doc/bridge_packages/flame_lottie/flame_lottie.md

```diff
@@ -0,0 +1,55 @@
+# flame_lottie
+
+This package allows you to load and add Lottie animations to your Flame game.
+
+> Lottie is a mobile library for Android and iOS that parses Adobe After Effects animations
```

Comment:
Maybe remove this quote since it only mentions android and iOS, and write a sentence about that it works on all platforms, to not confuse people.